.storybook/components/FluentDocsContainer.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
```

Comment:
This component was copied over from https://github.com/microsoft/fluentui/blob/master/apps/public-docsite-v9/src/DocsComponents/FluentDocsContainer.stories.tsx and will be removed once original component will be published as a part of a new package.

Reply:
can we aggregate these copy pasted things under a common folder ? `copied-from-fluentui-core/components` etc ?

more self explanatory then leaving comments per file I guess

also please open issue to track this TODO work. ty